connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -145,7 +171,26 @@ public Response listConnectors(
 
     @POST
     @Path("/")
-    @Operation(summary = "Create a new connector")
+    @Operation(
+            summary = "Create a new connector",
+            requestBody = @RequestBody(
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = CreateConnectorRequest.class)),
+                    required = true,
+                    description = "Name and configuration parameters of the new connector. All values in config should be strings."
+            ),
+            parameters = @Parameter(in = ParameterIn.QUERY, hidden = true, schema = @Schema(implementation = BooleanSchema.class))
+    )
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "201",
+                    description = "New connector created successfully",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ConnectorInfo.class))
+            ),
+            @ApiResponse(
+                    responseCode = "409",
+                    description = "Error while creating new connector. Either rabalance is in progress or the connector already exists"
```

Comment:
Typo in `rebalance`. Same a few more times below.
Also can we rephrase it to: `Either a rebalance is in progress or the connector already exists`?